align/schema/constraint.py

```diff
@@ -693,6 +690,27 @@ def pairs_validator(cls, value):
                         f"Incorrent symmetry pair {pair} in subckt {cls._validator_ctx().parent.parent.name}"
         return value
 
+    def check(self, checker):
+        for pair in self.pairs:
+            # TODO: This constraint is not fully hardened yet. Below only checks mirroring
+            if len(pair) == 2:
+                bvars = checker.iter_bbox_vars(pair)
+                for b1, b2 in itertools.pairwise(bvars):
+                    if self.direction == 'V':
+                        pd = 'sy'
+                        od = 'sx'
+                    else:
+                        pd = 'sx'
+                        od = 'sy'
+                    yield getattr(b1, pd) == getattr(b2, pd)
+                    if self.mirror:
+                        yield getattr(b1, od) != getattr(b2, od)
+                    else:
+                        yield getattr(b1, od) == getattr(b2, od)
```

Comment:
Looks like Mirror can either be True or False. What about don't care? This feels like more of a UserConstraint than a core constraint as it is constraining the search space.

Reply:
True is the preferred solution. False is useful if the manufacturing process has any shadowing effect. I suggest we defer don't care unless there is a motivating example.